src/EnergyPlus/HVACManager.cc

```diff
@@ -2288,56 +2186,39 @@ void ResetNodeData(EnergyPlusData &state)
 void UpdateZoneListAndGroupLoads(EnergyPlusData &state)
 {
 
-    // SUBROUTINE INFORMATION:
-    //       AUTHOR         Apparently someone who doesn't believe in documenting.
-    //       DATE WRITTEN   ???
-    //       MODIFIED       na
-    //       RE-ENGINEERED  na
-
-    // Using/Aliasing
-    using namespace DataHeatBalance;
-
     // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
     int ZoneNum;
     int ListNum;
     int GroupNum;
     int Mult;
 
-    auto &ZoneList = state.dataHeatBal->ZoneList;
-    auto &ZoneGroup = state.dataHeatBal->ZoneGroup;
-    auto &ZoneListSNLoadHeatEnergy = state.dataHeatBal->ZoneListSNLoadHeatEnergy;
-    auto &ZoneListSNLoadCoolEnergy = state.dataHeatBal->ZoneListSNLoadCoolEnergy;
-    auto &ZoneListSNLoadHeatRate = state.dataHeatBal->ZoneListSNLoadHeatRate;
-    auto &ZoneListSNLoadCoolRate = state.dataHeatBal->ZoneListSNLoadCoolRate;
-
```

Comment:
[src/EnergyPlus/HVACManager.cc:2306]:(style),[constVariable],Variable 'ZoneList' can be declared as reference to const
[src/EnergyPlus/HVACManager.cc:2307]:(style),[constVariable],Variable 'ZoneGroup' can be declared as reference to const
[src/EnergyPlus/HVACManager.cc:2308]:(style),[constVariable],Variable 'ZoneListSNLoadHeatEnergy' can be declared as reference to const
[src/EnergyPlus/HVACManager.cc:2309]:(style),[constVariable],Variable 'ZoneListSNLoadCoolEnergy' can be declared as reference to const
[src/EnergyPlus/HVACManager.cc:2310]:(style),[constVariable],Variable 'ZoneListSNLoadHeatRate' can be declared as reference to const
[src/EnergyPlus/HVACManager.cc:2311]:(style),[constVariable],Variable 'ZoneListSNLoadCoolRate' can be declared as reference to const